pkg/prometheus/resource_selector.go

```diff
@@ -1501,6 +1562,18 @@ func (rs *ResourceSelector) validateScalewaySDConfigs(ctx context.Context, sc *m
 		if _, err := rs.store.GetSecretKey(ctx, sc.GetNamespace(), config.SecretKey); err != nil {
 			return fmt.Errorf("[%d]: %w", i, err)
 		}
+
+		if err := validateProxyConfig(ctx, config.ProxyConfig, rs.store, sc.GetNamespace()); err != nil {
```

Comment:
Add historical omissions

Reply:
Move to other PR
ref: https://github.com/prometheus-operator/prometheus-operator/pull/7239